src/api/cpp/cvc5_types.h

```diff
@@ -28,15 +28,28 @@ namespace cvc5 {
  */
 enum UnknownExplanation
 {
+  /**
+   * Full satisfiability check required (e.g., if only preprocessing was
+   * performed).
+   */
   REQUIRES_FULL_CHECK,
+  /** Incomplete theory. */
   INCOMPLETE,
+  /** Time limit reached. */
   TIMEOUT,
+  /** Resource limit reached. */
   RESOURCEOUT,
+  /** Memory limit reached. */
   MEMOUT,
+  /** Solver was interrupted. */
   INTERRUPTED,
+  /** ??? */
```

Comment:
This corresponds to something from the optimization solver, I'm not sure what this means.

Reply:
I think that it can be removed like so #8518.